src/python/grpcio/grpc/_channel.py

```diff
@@ -906,11 +977,13 @@ def __init__(self, target, options, credentials):
           target: The target to which to connect.
           options: Configuration options for the channel.
           credentials: A cygrpc.ChannelCredentials or None.
+          wait_for_ready: A bool to enable wait-for-ready.
         """
         self._channel = cygrpc.Channel(
             _common.encode(target), _options(options), credentials)
         self._call_state = _ChannelCallState(self._channel)
         self._connectivity_state = _ChannelConnectivityState(self._channel)
+        self._initial_metadata_flags = _InitialMetadataFlags()
```

Comment:
I'm a little stuck on this....

Per [this gRFC comment](https://github.com/grpc/proposal/pull/112#issuecomment-435272949), I thought that we wanted the `wait_for_ready` flag to be per-call, rather than per-channel?

In that case, why are the `_initial_metadata` flags a property of the channel?